gherkin/php/composer.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "cucumber/gherkin",
+    "description": "Gherkin parser",
+    "author": "Cucumber Limited <cukes@googlegroups.com>",
+    "license": "MIT",
+    "type": "library",
+    "autoload": {
+        "psr-4": {
+            "Cucumber\\Gherkin\\": ["src/", "src-generated/"]
+        }
+    },
+    "require": {
+        "php": "^8.1",
+        "ext-mbstring": "*",
+        "cucumber/messages": "dev-main"
```

Comment:
This needs to reflect the most recent release, in tagged versions. How does that hook in to the build process?